pkg/webui/console/views/organization/index.js

```diff
@@ -71,30 +70,51 @@ const OrganizationInner = () => {
   const { orgId } = useParams()
   const organization = useSelector(selectSelectedOrganization)
   const name = organization.name || orgId
-  const dispatch = useDispatch()
   const siteName = selectApplicationSiteName()
 
   useBreadcrumbs('orgs.single', <Breadcrumb path={`/organizations/${orgId}`} content={name} />)
 
-  useEffect(
-    () => () => {
-      dispatch(stopOrganizationEventsStream(orgId))
+  const basePath = `/organizations/${orgId}`
+
+  const tabs = [
+    {
+      title: sharedMessages.members,
+      name: 'members',
+      link: basePath,
+      icon: IconCollaborators,
     },
-    [dispatch, orgId],
-  )
+    {
+      title: sharedMessages.apiKeys,
+      name: 'api-keys',
+      link: `${basePath}/api-keys`,
+      icon: IconKey,
+    },
+    {
+      title: sharedMessages.settings,
+      name: 'general-settings',
+      link: `${basePath}/general-settings`,
+      icon: IconGeneralSettings,
+    },
+  ]
 
   return (
-    <React.Fragment>
+    <>
       <IntlHelmet titleTemplate={`%s - ${name} - ${siteName}`} />
+      <OrganizationHeader org={organization} />
+      <Tabs
+        className="w-full"
```

Comment:
```suggestion
```
Let's use a different non-stretched layout here, as it looks pretty odd with only 3 entries.